### ISSUE TITLE
New version to test - 1.1.0

### ISSUE THREAD

**Issue by dodgyrabbit** [timothycrosley/streamdeck-ui#115]
I've published a **test version [1.1.0](https://test.pypi.org/project/streamdeck-ui/)** on test.pypi.org for those of you interested to give it a run and provide feedback before it is published on the main repository.

The change log is [here](https://github.com/timothycrosley/streamdeck-ui/blob/dim/CHANGELOG.md)

### How to upgrade
This assumes you have version 1.0.7 already working on your system. You can confirm your installed version with:
``` console
$ pip3 show streamdeck-ui
Name: streamdeck-ui
Version: 1.0.7
```

Uninstall your current version
``` console
$ pip3 uninstall streamdeck-ui 
```

Install the latest version from the **test.pypi.org** server.
``` console
$ pip3 install -i https://test.pypi.org/simple/ streamdeck-ui==1.1.0 --user
```
You can confirm with
``` console
$ pip3 show streamdeck-ui
Name: streamdeck-ui
Version: 1.1.0 🎉
```
Launch streamdeck 
``` console
$ streamdeck
```
The brightness slider has moved to a settings dialog.
![1 1 0](https://user-images.githubusercontent.com/518187/114318404-c1ce4180-9ada-11eb-8d54-29678a1794c9.png)

Under settings you can configure auto dim.

![fade](https://user-images.githubusercontent.com/518187/114319086-fbed1280-9add-11eb-91a8-044d7fcf6f9d.gif)

The display will dim after the configured timeout. Pressing any of the buttons will return the screen to the configured brightness (but not trigger the button action). Once the display is "woken up", you can press buttons as before.

You can configure the Stream Deck to dim to 0% after a couple of predefined time intervals, which will suit most.
If you really want something else, you can modify the number of seconds in the `~/.streamdeck_ui.json` file:
``` json
"display_timeout": 60
```

### How to downgrade
Remove the test version
```
$ pip3 uninstall streamdeck-ui
```
Install the latest from the **main** repository. 
``` console
$ pip3 install streamdeck-ui --user
```

If you are running into issues, please open a separate issue and provide as much detail as possible. Steps you took, output, your operating system version, streamdeck version, screenshots - whatever can help. Thanks!

**Comment by Mxtreme1** [comment]
Indeed works very well, thank you!
A dim option to a lower brightness might still be of use though.

P.S. You forgot to blur your device number in the second image.

**Comment by dodgyrabbit** [comment]
Thanks for the feedback. I guess the cats out the bag now :)

Maybe the next feature should be giving a device a name.

**Comment by dodgyrabbit** [comment]
I've published v1.1 to pypi.org. Thanks to everyone who tried it!
